File: src/main/java/com/rwbymod/z_proxy/ClientProxy.java
package com.rwbymod.z_proxy;

import com.rwbymod.block.BlockRegistry;
import com.rwbymod.client.RenderRegistry;
import com.rwbymod.item.ItemRegistry;

public class ClientProxy extends CommonProxy {

	public void preInit() {
		super.preInit();
	}
	
	public void init() {
		super.init();
		RenderRegistry.init();
	}
	
	public void postInit() {
		super.postInit();
	}

}

File: src/main/scala/com/rwbymod/z_proxy/ClientProxy.scala
package com.rwbymod.z_proxy

import com.rwbymod.registry.RenderRegistry

class ClientProxy extends CommonProxy {

  override def preInit = { super.preInit }

  override def init = {
    super.init
    RenderRegistry.init
  }

  override def postInit = { super.postInit }

}